dnfdaemon-server/services/rpm/rpm.cpp

```diff
@@ -24,14 +24,20 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
 
 #include <libdnf/rpm/package_query.hpp>
 #include <libdnf/rpm/package_set.hpp>
+#include <sdbus-c++/Message.h>
 #include <sdbus-c++/sdbus-c++.h>
 
 #include <iostream>
 #include <string>
+#include <vector>
 
 
 void Rpm::dbus_register() {
     auto dbus_object = session.get_dbus_object();
+    dbus_object->registerMethod(
+        dnfdaemon::INTERFACE_RPM, "distro-sync", "a{sv}", "", [this](sdbus::MethodCall call) -> void {
```

Comment:
Method names "Must only contain the ASCII characters "[A-Z][a-z][0-9]_" and may not begin with a digit." (see Member names section in  https://dbus.freedesktop.org/doc/dbus-specification.html)
We should either use one word `distrosync` or underscore `distro_sync`.